cm/config.py

```diff
@@ -16,186 +16,237 @@
-        elif cloud_name == "hpcloud":
-            instance_types = [
-                ("", "Same as Master"),
-                ("standard.xsmall", "Extra Small"),
-                ("standard.small", "Small"),
-                ("standard.medium", "Medium"),
-                ("standard.large", "Large"),
-                ("standard.xlarge", "Extra Large"),
-                ("standard.2xlarge", "Extra Extra Large"),
-            ]
-        elif cloud_name == "ict-tas":
-            instance_types = [
-                ("", "Same as Master"),
-                ("m1.small", "Small"),
-                ("m1.medium", "Medium"),
-                ("m1.xlarge", "Extra Large"),
-                ("m1.xxlarge", "Extra Extra Large"),
-            ]
+            return instance_types
         else:
-            instance_types = DEFAULT_INSTANCE_TYPES
-        self.instance_types = instance_types
+            keys = [key for key in DEFAULT_INSTANCE_TYPES.keys() if key in self.cloud_name]
+            if keys:
+                return DEFAULT_INSTANCE_TYPES.get(keys[0])
 
+    @property
+    def cloudman_repo_url(self):
+        return self.get("CM_url", "https://bitbucket.org/galaxy/cloudman/commits/all?page=tip&search=")
 
-def get_database_engine_options(kwargs):
-    """
-    Allow options for the SQLAlchemy database engine to be passed by using
-    the prefix "database_engine_option_".
-    """
-    conversions = {
-        'convert_unicode': string_as_bool,
-        'pool_timeout': int,
-        'echo': string_as_bool,
-        'echo_pool': string_as_bool,
-        'pool_recycle': int,
-        'pool_size': int,
-        'max_overflow': int,
-        'pool_threadlocal': string_as_bool
-    }
-    prefix = "database_engine_option_"
-    prefix_len = len(prefix)
-    rval = {}
-    for key, value in kwargs.iteritems():
-        if key.startswith(prefix):
-            key = key[prefix_len:]
-            if key in conversions:
-                value = conversions[key](value)
-            rval[key] = value
-    return rval
-
-
-def configure_logging(config, user_data={}):
+    @property
+    def ignore_unsatisfiable_dependencies(self):
+        return self.get("ignore_unsatisfiable_dependencies", False)
+
+    @property
+    def web_thread_count(self):
+        return self.get("web_thread_count", 3)
+
+    @property
+    def info_brand(self):
+        return self.get("brand", "")
+
+    @property
+    def info_wiki_url(self):
+        return self.get("wiki_url", "http://g2.trac.bx.psu.edu/")
+
+    @property
+    def info_bugs_email(self):
+        return self.get("bugs_email", "mailto:galaxy-bugs@bx.psu.edu")
+
+    @property
+    def info_blog_url(self):
+        return self.get("blog_url", "http://g2.trac.bx.psu.edu/blog")
+
+    @property
+    def info_screencasts_url(self):
+        return self.get("screencasts_url", "http://main.g2.bx.psu.edu/u/aun1/p/screencasts")
+
+
+# TODO: REFACTOR - This method doesn't belong here
+def configure_logging(config):
```

Comment:
Just wanted to flag that it would be nice to stick this method somewhere else, since it doesn't seem particularly relevant.